src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/integration/visits/VisitsMigrationIntTest.kt

```diff
@@ -50,7 +54,8 @@ class VisitsMigrationIntTest : IntegrationTestBase() {
 
     @Test
     internal fun `will start processing pages of visits`() {
-      nomisApi.stubGetVisitsInitialCount(23_045)
+      nomisApi.stubGetVisitsInitialCount(86)
+      nomisApi.stubMultipleGetVisitsCounts(totalElements = 86, pageSize = 10)
```

Comment:
test would take way too long to process 23 thousand visits - so just do 86 and reduce page size to 10 for tests